### ISSUE TITLE
Update all account create examples to use an ECDSA public key and set the corresponding EVM address derived from it

### ISSUE THREAD

**Issue by SimiHunjan** [hiero-ledger/hiero-docs#838]
### Problem

To reduce confusion about the key type to use when creating accounts, we need to update all examples to use ECDSA public keys and set the corresponding EVM address

### Solution

- Review all places where the docs show how to create an account
- Update all places to demonstrate the following:

```
const transaction = new AccountCreateTransaction()
     .setKey(ecdsaPublicKey)
     .setAlias(ecdsaPublicKey.toEvmAddress())
     .setInitialBalance(new Hbar(1))
```

**Comment by SimiHunjan** [comment]
Also to note that if the alias is set that it is immutable.

If the user is intending to rotate keys/update keys at a future point then the alias should should not be set upon initial creation, but after the keys are updated and will not require future updates.

**Comment by theekrystallee** [comment]
resolved [42d0a5c](https://github.com/hashgraph/hedera-docs/commit/42d0a5cc5a784d537c76414a56d433f64398ac65)
